docs/topics/review-reports.adoc

```diff
@@ -297,7 +297,7 @@ This report provides the list of rules that executed when running the {ProductSh
 .Rule Providers Execution Overview
 image::report-jee-example-ruleprovider.png[Rule Provider Execution Overview]
 
-[id='review-the-report-freemarker-report_{context}']
+[id="review-the-report-freemarker-report_{context}"]
 == Used FreeMarker Functions and Directives

```

Comment:
I don't know the context but is "Used" really the correct word? "Functions" and "Directives" should be lower case.